src/lsdb/loaders/hipscat/hipscat_catalog_loader.py

```diff
@@ -0,0 +1,98 @@
+from typing import List, Tuple
+
+import dask.dataframe as dd
+import hipscat as hc
+import pyarrow
+from hipscat.pixel_math import HealpixPixel
+from hipscat.pixel_math.hipscat_id import HIPSCAT_ID_HEALPIX_ORDER
+
+from lsdb import io
+from lsdb.catalog.catalog import Catalog, DaskDFPixelMap
+from lsdb.loaders.hipscat.hipscat_loading_config import HipscatLoadingConfig
+
+
+# pylint: disable=R0903
```

Comment:
Can we either 1) use the human-readable pylint name or 2) adjust the min number of public methods in the .pylintrc? Again, something that can wait for a follow-up.

Reply:
Updated, thanks